lib/airbrake/capistrano/tasks.rb

```diff
@@ -6,7 +6,8 @@
   namespace :airbrake do
     desc "Notify Airbrake of the deploy"
     task :deploy do
-      on roles(:all) do
+      role = roles(:all, select: :primary).first || roles(:all).first
```

Comment:
```ruby
roles(:all).first
``` 
would likely work fine, I think the point is we want it only notifying on 1 server, not all of them.  It was doing it that way at one point, however I'm not sure when it changed, I just noticed when I realized none of our projects were notifying Airbrake anymore, something that must have been missed in our upgrade to Capistrano 3.x.  Choosing the primary server to do the task seems to be the standard, so that's why I went with that.  I can however change it to just be `roles(:all).first` if that's what you would prefer.  Let me know and I'll update the pull request.